### DISCUSSION TITLE
Usage policy enforcement

### DISCUSSION THREAD

**Discussion by GlennDigipolis** [osm-search/Nominatim#2819] [Q&A]
Hello,

I'm currently debugging an issue where I expect that we passed the usage limit.
How does nominatim handle requests from sender/email that has passed this limit.
Does it send a response with error code 429 (Too Many Requests)?
Is there anyway to see if we reached this limit?

Thanks,
Glenn

**Comment by lonvia** [comment]
nominatim.openstreetmap.org will send 429 as a warning before blocking you completely for a period of time.

As a rule of thumb: if you have to ask what exactly happens with the usage limits, then nominatim.openstreetmap.org is likely not the right choice for your use case.
